models/User.js

```diff
@@ -5,9 +5,9 @@ const userSchema = new Schema({
 	isOwner: Boolean,
 	isPlayer: Boolean,
 	isAdmin: Boolean,
-	ign: String,
 	elo: Number,
-	team: String
+	team: String,
+  accounts: [Schema.Types.ObjectId]
```

Comment:
definitely everything should be ObjectIds and then use population to return the objects in the query for the user.